state/stateConsensus.go

```diff
@@ -87,8 +87,11 @@ func (s *State) Process() (progress bool) {
 		if now-s.StartDelay > s.StartDelayLimit {
 			if s.DBFinished == true {
 				s.RunLeader = true
-				s.StartDelay = now // Reset StartDelay for Ignore Missing
-				s.IgnoreMissing = true
+				if !s.IgnoreDone {
+					s.StartDelay = now // Reset StartDelay for Ignore Missing
```

Comment:
Correct. IgnoreDone is not initialized to a particular value _except_ right here, though (so it is initially going to be false as things boot up). And if IgnoreDone _is_ true, then that should indicate that we don't need to reset the state's StartDelay value (since we don't need to go through the "Ignore interval"; that phase should be done).